core/arch/arm/include/kernel/thread_defs.h

```diff
@@ -31,5 +31,6 @@
 #define THREAD_FLAGS_COPY_ARGS_ON_RETURN	(1 << 0)
 #define THREAD_FLAGS_IRQ_ENABLE			(1 << 1)
 #define THREAD_FLAGS_EXIT_ON_IRQ		(1 << 2)
+#define THREAD_FLAGS_NONNATIVE_INTR_ENABLE	(1 << 3)
```

Comment:
Just don't want to impact any existing code (public or private) which uses the THREAD_FLAGS_IRQ_ENABLE.